UI/GalaxySetupWnd.cpp

```diff
@@ -615,7 +615,7 @@ void GalaxySetupPanel::CompleteConstruction() {
     m_monster_freq_label = GG::Wnd::Create<CUILabel>(UserString("GSETUP_MONSTER_FREQ"), GG::FORMAT_RIGHT, GG::INTERACTIVE);
     m_monster_freq_label->SetBrowseModeTime(GetOptionsDB().Get<int>("ui.tooltip.delay"));
     m_monster_freq_label->SetBrowseText(UserString(GetOptionsDB().GetDescription("setup.monster.frequency")));
-    m_monster_freq_list = GG::Wnd::Create<CUIDropDownList>(5);
+    m_monster_freq_list = GG::Wnd::Create<CUIDropDownList>(8);
```

Comment:
stop adding unrelated features to this PR. it only adds delay - i wont be able to spend any time reviewing soon

Reply:
my bad. misplaced commit. has its own pr. i overlooked this commit and is about to be reverted.